src/backend.c

```diff
@@ -3558,6 +3504,7 @@ int ocl_init (hashcat_ctx_t *hashcat_ctx)
   HC_LOAD_FUNC (ocl, clCreateProgramWithBinary, OCL_CLCREATEPROGRAMWITHBINARY,  OpenCL, 1);
   HC_LOAD_FUNC (ocl, clCreateProgramWithSource, OCL_CLCREATEPROGRAMWITHSOURCE,  OpenCL, 1);
   HC_LOAD_FUNC (ocl, clEnqueueCopyBuffer,       OCL_CLENQUEUECOPYBUFFER,        OpenCL, 1);
+  HC_LOAD_FUNC (ocl, clEnqueueFillBuffer,       OCL_CLENQUEUEFILLBUFFER,        OpenCL, -1);
```

Comment:
Is it really -1? I think we used 0 for expected unavailability cases.

Reply:
If value is -1, we are ignoring the problem. But if it's 1, we would report error "missing from shared library" and return with -1. Then again if it's not -1 or 1, for example 0, we would report error "missing from shared library" and return with 0. Early return would result that the rest of symbols would not be loaded. Zero would work if this would be the last and only optional function ever. I decided to silently ignore it and solve the problem with a workaround.

Workaround should only be needed with OpenCL before version 1.2, but then again I didn't want to leave a possible new bug to be fixed later as workaround is quite small and easy. Same idea was used before.